aws-auth-configmap.tf

```diff
@@ -4,13 +4,7 @@ resource "kubernetes_config_map" "aws_auth" {
   metadata {
     name      = "aws-auth"
     namespace = "kube-system"
-    labels = merge(
-      {
-        "app.kubernetes.io/managed-by" = "Terraform"
-        "terraform.io/module"          = "terraform-aws-eks-blueprints"
-      },
-      var.aws_auth_additional_labels
-    )
```

Comment:
We can keep the `terraform.io/module`  label for `aws-auth`. We only need to remove `app.kubernetes.io/managed-by` label from all the all the Add-ons supported by Helm Charts to ensure this is going to cause an issue when ArgoCD is enabled for these add-ons